src/libstore/build/derivation-goal.cc

```diff
@@ -9,7 +9,7 @@
 #include "nix/util/util.hh"
 #include "nix/util/compression.hh"
 #include "nix/store/common-protocol.hh"
-#include "nix/store/common-protocol-impl.hh"
+#include "nix/store/common-protocol-impl.hh" // Don't remove is actually needed
```

Comment:
Interestingly, this one was needed for some C++ templates.
I'm not clear why cland does not pick it up -- I'm not up to date on my C++ knowledge.